rust/shalom_dart_codegen/dart_tests/test/custom_scalar_arguments/operations.graphql

```diff
@@ -0,0 +1,56 @@
+
+mutation UpdatePointCoordsNonNull($coords: Point!) {
+    updatePointCoords(coords: $coords) {
+        coords,
+        name,
+        id
+    }
+}
+
+mutation UpdatePointCoordsOpt($coords: Point) {
+    updatePointCoordsOpt(coords: $coords) {
+        coords,
+        name,
+        id
+    }
+}
+
+mutation UpdatePointCoordsMaybe($coords: Point) {
+    updatePointCoordsMaybe(coords: $coords) {
+        coords
+        name
+        id
+    }
+}
+
+mutation UpdatePointWithInputCoordsMaybe($pointData: PointUpdateCoordsMaybe!) {
+    updatePointWithInputCoordsMaybe(pointData: $pointData) {
+        coords
+        name
+        id
+    }
+}
+
+mutation UpdatePointWithInputNonNull($pointData: PointDataInput!) {
+    updatePointWithInput(pointData: $pointData) {
+        coords,
+        name,
+        id
+    }
+}
+
+mutation UpdatePointWithInputCoordsOpt($pointData: PointUpdateCoordsOpt!) {
+    updatePointWithInputCoordsOpt(pointData: $pointData) {
+        coords,
+        name,
+        id
+    }
+}
+
+query GetPointWithDefaultCoords($id: ID!, $coords: Point = "POINT (0,0)") {
```

Comment:
```suggestion
query UpdatePointCoordsOpt($id: ID!, $coords: Point = "POINT (0,0)") {
```
and move it near maybe

Reply:
on line 10 we already have UpdatePointCoordsOpt would'nt that create a conflict, we will have two operations named UpdatePointCoordsOpt(one mutation, one query)

Reply:
don't resolve stuff before you commit.

Reply:
Opt is ok. Maybe is for opt with no default args, check the reasoning behind it [here](https://github.com/nrbnlulu/shalom/pull/29#pullrequestreview-2816894849). essentially every optional argument that has no default argument (be it null or value) is wrapped in a `Maybe` this allows to distinguish  between an absence of a value and it being null.